src/resource.js

```diff
@@ -124,6 +152,16 @@ class Resource extends BaseResource {
     }, {})
     return new ValidationError(`${this.name()} validation failed`, errors)
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  convertMongooseObject(mongooseObj) {
```

Comment:
I would change that to static method (to fix `class-methods-use-this`) and change the name to: `stringifyId`)